metricshub-snmpv3-extension/src/main/java/org/sentrysoftware/metricshub/extension/snmpv3/SnmpV3Extension.java

```diff
@@ -105,15 +93,14 @@ public Set<Class<? extends Criterion>> getSupportedCriteria() {
 	}
 
 	@Override
-	public void checkProtocol(TelemetryManager telemetryManager) {
+	public boolean checkProtocol(TelemetryManager telemetryManager) {
 		// Retrieve the hostname
 		final String hostname = telemetryManager.getHostConfiguration().getHostname();
 
-		// Retrieve the host endpoint monitor
-		final Monitor hostMonitor = telemetryManager.getEndpointHostMonitor();
+		log.info("Hostname {} - Performing protocol health check.", hostname);
```

Comment:
`log.info("Hostname {} - Performing {} protocol health check.", hostname, getIdentifier());`

Reply:
Move the `log.info`  after the (snmpV3Configuration == null) check please.